### ISSUE TITLE
Missing Product Attributes

### ISSUE THREAD

**Issue by simonmaass** [DivanteLtd/magento2-vsbridge-indexer#293]
I am missing the alt text attributes on images since update...

![image](https://user-images.githubusercontent.com/32599710/84370918-3691d000-abd9-11ea-98b1-2cc56ee12a29.png)

Like here "Sneaker"

**Comment by afirlejczyk** [comment]
Hi, @simonmaass 
Which update? After updating module? If so, what version do you use previously, and which version do you use now? 

With the latest version, you can now choose which attribute to export in system configuration. I would try to use the "All attributes" option - and see if it help.

"Alt text" - this a custom attribute or some default Magento attribute? (from sample data)

**Comment by afirlejczyk** [comment]
Hi @simonmaass please check if it related to issue #284? Do you have wrong value or no value at all in ES?
